application/view/omeka/admin/item-set/show.phtml

```diff
@@ -13,30 +13,30 @@ $viewItemsLink = $this->url(
 if ($owner = $itemSet->owner()) {
     $ownerText = $escape($itemSet->owner()->name());
 } else {
-    $ownerText = $this->translate('[no owner]');
+    $ownerText = $translate('[no owner]');
```

Comment:
Am I just missing where `$translate` is defined in most of these? Is it just "leaking" in from some previous view?

Reply:
This is a small optimization, `translate` is used very often, so it should be used and set in most of the theme file. I check if it is missed in one file. Fixed in next commit.